src/checkpoints/checkpoints.cpp

```diff
@@ -249,6 +249,7 @@ namespace cryptonote
     ADD_CHECKPOINT2(2325000, "f61261994b368700f0cbbfb4477433fa36a3c7537908ab2d1a06ac2987cc8b01", "0x154bceeffaff847");
     ADD_CHECKPOINT2(2430000, "5f7f4273eb6be024df3c4311a8d28482220a253190efa1719f35b7782587740c", "0x1c3f14ba9b0bebc");
     ADD_CHECKPOINT2(2478000, "692fc889f5328c9cfe47e9846ca2bb3d61d82dbeb37077e90e0f8c36b8fc0c84", "0x1fbb38d0aae41a6");
+    ADD_CHECKPOINT2(2606000, "41a212fe33f9317a702a163e4c45b6295a9cc303745fe21bfe499a5e39b119c7", "0x29e2348ffb1e034");
```

Comment:
Got the same hash using:
`curl http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_block_header_by_height","params":{"height":2606000}}' -H 'Content-Type: application/json'`

Reply:
I just realized it's even easier to check with https://xmrchain.net/block/41a212fe33f9317a702a163e4c45b6295a9cc303745fe21bfe499a5e39b119c7
```
Block hash (height): 41a212fe33f9317a702a163e4c45b6295a9cc303745fe21bfe499a5e39b119c7 (2606000)
```

Reply:
Too easy :)